Fix/qa feedback select

jhossyd3v opened the pull request:
Includes: 
- Fix issue 103: Add plugin nice-select2, implement plugin and add styles for options and scrollbar
- Fix issue 119: Add missing text on footnote of adults tab on Hypophosphatasia page
- Fix issue 120: Change word training for support in injection-site-reactions section on dosing page
- Fix calculations of ISI scrolling that were affected by changes on internal-navigation

Commit:
fix: :sparkles: Fix issue 103: Styles of select on rep modal

Add plugin nice-select2, implement plugin on modal selects, add styles for options and scrollbar

Commit:
fix: :speech_balloon: Fix issue 119: Add missing text on footnote, adults tab on Hypophosphatasia page

Add missing text on footnote, adults tab on Hypophosphatasia page

Commit:
fix: :speech_balloon: Fix issue 120: Change word training for support, injection-site-reactions section, dosing page

Change word training for support, injection-site-reactions section, dosing page

Commit:
Merge branch 'develop' of https://github.com/HackMort/strensiq-hcp into fix/qa-feedback-select

Commit:
fix: :bug: Fix calculations of ISI scrolling after changes on navigation

Fix calculations of ISI scrolling after changes on navigation

Commit:
fix: :lipstick: Fix issue with not hover items on select

Add custom styles to overwrite  :not(:hover) styles by default on nice-select2 library

HackMort commented on package.json:
```diff
@@ -12,7 +12,8 @@
     "@fancyapps/ui": "5.0.14",
-    "astro": "2.1.8"
+    "astro": "2.1.8",
+    "nice-select2": "^2.1.0"
```
Please remove the carot symbol

HackMort commented on src/components/RequestARepModalForm.astro:
```diff
@@ -302,6 +302,10 @@
     </div>
   </div>
 </div>
+<style>
```
Instead of this, do the import in the Layout at the end of the day, we need it on every single plage, you could do the same for the script.

HackMort replied:
Something like:

```
import "/node_modules/nice-select2/src/scss/nice-select2.scss";
```

HackMort reviewed (changes requested):
The functionality is there but we need to fix a couple of css issues I found:
![image](https://github.com/HackMort/strensiq-hcp/assets/6466646/10008ed1-8535-487b-8cc7-1db9c0b83ffe)
As you can see there's a border inside "our element border", also the text is showing to low, should be vertical centered, the angle-down icon it's not centered either. This happens on both selects.

Other than that, I think we

Commit:
fix: :lipstick: Fix borders inside select and center alingment

Remove borders for nice-select2 library and fix vertical aligment for text and arrow icon

Commit:
fix: :wrench: Remove carot symbol for nice-select2 dependency declaration in package.json

Remove carot symbol for nice-select2 dependency declaration in package.json

Commit:
fix: :construction: Change the way of import css file of nice-select2

Change the way of import css file of nice-select2

Commit:
fix: :wrench: Remove carot from package-lock.json on import of library nice-select2

Remove carot from package-lock.json on import of library nice-select2

HackMort committed:
build: :rocket: Update to use nice-select2 library with statics files instead of NPM package.
